iree/modules/tensorlist/BUILD

```diff
@@ -20,6 +20,13 @@ package(
     licenses = ["notice"],  # Apache 2.0
 )
 
+iree_cmake_extra_content(
+    content = """
+if(${IREE_BUILD_COMPILER} AND ${IREE_BUILD_TESTS})
```

Comment:
Instead of IREE_BUILD_TESTS, you can add `testonly = True` to the testonly rule. Then it will only get built if tests are enabled because of https://github.com/google/iree/blob/4ca12072652be6785075e8e6e48052347f9de784/build_tools/cmake/iree_bytecode_module.cmake#L46-L48

Reply:
I believe `testonly = True` isn't currently supported for iree_bytecode_module in bazel_to_cmake, so can I leave the check for IREE_BUILD_TESTS for now and update this and bazel_to_cmake in a followup PR?

Reply:
Ah you're right. SG, thanks :-)